Sources/SourceKitBazelBSP/RequestHandlers/SKOptions/CompilerArgumentsProcessor.swift

```diff
@@ -154,6 +88,15 @@ enum CompilerArgumentsProcessor {
         while index < count {
             let arg = rawArguments[index]
 
+            // Skip worker and swiftc
+            if arg.hasSuffix("/worker"),
+                index + 1 < count,
+                rawArguments[index + 1] == "swiftc"
+            {
+                index += 2
+                continue
+            }
```

Comment:
Is it possible to instead keep the previous logic where the first few args would be skipped based on the platform? If you see the example Obj-C output on the test below, this resulted in some wrong arguments slipping into the result.